module-user/src/main/kotlin/com/example/user/application/port/in/SelectStreamerUserUseCase.kt

```diff
@@ -0,0 +1,7 @@
+package com.example.user.application.port.`in`
+
+import com.example.user.domain.model.StreamerUser
+
+interface SelectStreamerUserUseCase {
+    fun selectPendingStreamers() : List<StreamerUser>
+}
```

Comment:
```suggestion
interface FindStreamerUserUseCase {
    fun findPendingStreamers() : List<StreamerUser>
}
```


> 조회 기능에 대한 네이밍에 select라는 prefix는 적절하지 않다고 생각합니다.
- query에 익숙한 사람은 select가 조회에 직관적으로 연결됩니다. 저도 그렇습니다.
- 하지만 query에 익숙하지 않은 사람은 select가 `조회`보다는 `선택`하는 액션으로 보일 수도 있습니다.
    - `스트리머를 조회해라`라는 액션을 POST 메서드로 만들면서 그것에 대한 네이밍을 select로 하는 경우가 생길 수도 있다고 생각합니다.
    - 물론 `선택`은 `choose`라는 대체어가 있긴 합니다.
    - 하지만 `조회` 역시 `find`나 `search`같은 대체어가 있음으로 좀 더 직관적인 언어를 사용하면 좋을 것 같습니다.

Reply:
굿 코멘트네요